intel_nn_hal/GnaPreparedModel.cpp

```diff
@@ -113,6 +131,7 @@ bool GnaPreparedModel::initialize() {
     mBuilderModel = new IRBuilder::ModelBuilder();
     mBuilderModel->initializeBuilder();
 
+    time_point irbuild_start = now();
```

Comment:
Let us  either remove the profiling data from main branch or move it under #define so that we can turn on only when required